Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -1048,9 +1057,9 @@ def validate_range(range: Optional[str]) -> tuple[Optional[float], Optional[floa
         range_without_spaces = range.strip()
         nums_list = range_without_spaces.split("-")
         if len(nums_list) < 2:
-            raise DemistoException('Please specify valid vprScoreRange')
+            raise DemistoException('Please specify valid vprScoreRange. For example: 3.5-5.5.')
```

Comment:
There is no documentation for this?
This is an example, but can I put here a bigger number? or lower? 
What the docs states?

Reply:
I saw in the yml - VPR values range are 0.1-10.0
You need to write this as well